cli/internal/hashing/package_deps_hash_test.go

```diff
@@ -239,24 +239,43 @@ func TestGetPackageDeps(t *testing.T) {
 
 	repoRoot := fs.AbsolutePathFromUpstream(t.TempDir())
 	myPkgDir := repoRoot.Join("my-pkg")
+
+	// create the dir first
+	err := myPkgDir.MkdirAll()
+	assert.NilError(t, err, "CreateDir")
+
+	// create file 1
 	committedFilePath := myPkgDir.Join("committed-file")
-	err := committedFilePath.EnsureDir()
-	assert.NilError(t, err, "EnsureDir")
 	err = committedFilePath.WriteFile([]byte("committed bytes"), 0644)
 	assert.NilError(t, err, "WriteFile")
+
+	// create file 2
 	deletedFilePath := myPkgDir.Join("deleted-file")
 	err = deletedFilePath.WriteFile([]byte("delete-me"), 0644)
 	assert.NilError(t, err, "WriteFile")
+
+	// create file 3
 	nestedPath := myPkgDir.Join("dir", "nested-file")
 	assert.NilError(t, nestedPath.EnsureDir(), "EnsureDir")
 	assert.NilError(t, nestedPath.WriteFile([]byte("nested"), 0644), "WriteFile")
+
+	// create a package.json
+	packageJSONPath := myPkgDir.Join("package.json")
+	err = packageJSONPath.WriteFile([]byte("{}"), 0644)
+	assert.NilError(t, err, "WriteFile")
```

Comment:
Did I miss seeing this last time because of single commit mode or inattentiveness? I'm sorry.